packages/langchain-graph-retriever/src/langchain_graph_retriever/adapters/open_search.py

```diff
@@ -147,6 +147,7 @@ def _get(
             },
             index=self.vector_store.index_name,
             _source_includes=["text", "metadata", "vector_field"],
+            size=len(ids),
```

Comment:
bug found that open_search returns max 10 rows by default.

Reply:
Is there a regression test for this?